### PR TITLE
Introduce YAML to configure and templatize the deployment script

### LINKED ISSUE #12: Create a config driven approach to "personalize" the app for different jamaats
## Problem

The website was originally designed for SF jamaat. References to sf jamaat in the UI, deploy script, etc. need to be replaceable with plug-able values for any other jamaat.

These must not include secrets like dbhost, dbusername, dbpassword, dbname, etc. Secrets must be accepted at runtime as part of the build/deploy script.

## Possible Solutions

1. Create a constants file for every jamaat that can be provided as a parameter to the build/deploy script
2. Create a single constants file and each jamaat that chooses to deploy this application will create a fork of this repo and customize the constants file for their needs. To keep up-to-date with the latest features, they can merge from this repo from time-to-time and handle conflicts.

### PR DESCRIPTION
**Pull request by lakhia**
Created an example config/example.yaml as a starter configuration
Update README with new deploy instructions
Replaced all SF Jamaat specific words with templates

closes #12

### CODE REVIEW

File: app/plan.html [hunk 1]
```diff
@@ -5,9 +5,9 @@ <h3>Plan Your Event</h3>
 <br>
 
 <ul>
-    <big>Submit your <a href="https://form.jotform.com/222676794136163">raza
+    <big>Submit your <a href="PLANNING_URL">raza
         request</a>. For questions, please contact <a
-     href="mailto:secretary&#64;sfjamaat.org">our jamaat secretary</a>.
+     href="mailto:SECRETARY_EMAIL">our jamaat secretary</a>.
```
Comment (vizkid2005) [inline app/plan.html:10]: nit: Under 12 Umoor framework. This should now be Umoor FMB coordinator.

File: deploy.pl [hunk 2]
```diff
@@ -69,6 +70,14 @@ sub html {
...
     open OUT, ">$_.backup" or die "Cannot open $!";
     while ($line = <IN>) {
+        # Template values
+        $line =~ s/APP_NAME/$config->{'app'}->{'name'}/;
+        $line =~ s/FEEDBACK_URL/$config->{'links'}->{'feedback'}/;
+        $line =~ s/PLANNING_URL/$config->{'links'}->{'planning'}/;
+        $line =~ s/ADMIN_EMAIL/$config->{'email'}->{'admin'}/;
+        $line =~ s/CONTACT_EMAIL/$config->{'email'}->{'contact'}/;
+        $line =~ s/SECRETARY_EMAIL/$config->{'email'}->{'secretary'}/;
+
```
Comment (vizkid2005) [inline deploy.pl:80]: I am liking perl for semantics like these.

**Review by vizkid2005 — CHANGES_REQUESTED** [review]
Please update from contact from Jamaat secretary to FMB coordinator

File: app/plan.html (hunk 1 above)
Comment (lakhia) [reply app/plan.html:10]: Okay. So let's make title of the role also a template variable. Sound good?

**Comment by vizkid2005** [comment]
You can also link your change to this issue https://github.com/lakhia/rsvp-website/issues/12

Your PR is the solution to this issue

**Review by vizkid2005 — APPROVED** [review]
Ship it

**Pull request merged**